posthog/temporal/batch_exports/spmc.py

```diff
@@ -1090,7 +1090,7 @@ def compose_filters_clause(
         values=values or {},
         modifiers=HogQLQueryModifiers(materializationMode=MaterializationMode.DISABLED),
     )
-    context.database = create_hogql_database(team.id, context.modifiers)
+    context.database = create_hogql_database(team=team, modifiers=context.modifiers)
```

Comment:
Highly suggest making this param change in a separate PR to make this one more reviewable

Reply:
I agree. I've decided to split this PR into several PRs, this will make reviewing (and merging) this much easier.